### DISCUSSION TITLE
Teen Nick

### DISCUSSION THREAD

**Discussion by longusboroungus** [iptv-org/community#774] [Q&A]
### Channel Name

Teen Nick

### Channel ID

TeenNickEast.us

### Website

http://www.teennick.com/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19756
